CONTRIBUTING.md

```diff
@@ -4,27 +4,34 @@ As mentioned in the [README](https://github.com/haskell/containers/blob/master/R
 
 For proposing API changes/enhancements, please follow the [guidelines outlined on the Haskell Wiki](https://wiki.haskell.org/Library_submissions#Guide_to_proposers). Especially note that all API changes/enhancements should be discussed on libraries@haskell.org mailing list.
 
-## Building and testing
+## Building, testing, and benchmarking
 
-Building and testing the containers package can be done using either `cabal-install` or `stack`.
+Building, testing, and benchmarking the containers package can be done using either `cabal-install` or `stack`.
 
 ### With cabal-install
 
+Minimum cabal version: 1.24
+
 ```
 cabal sandbox init
 cabal install --only-dependencies
-cabal install 'test-framework >= 0.3.3' 'test-framework-quickcheck2 >= 0.2.9' 'QuickCheck >= 2.4.0.1' 'ChasingBottoms' 'HUnit' 'test-framework-hunit'
-cabal configure -v2 --enable-tests
+cabal install 'test-framework >= 0.3.3' 'test-framework-quickcheck2 >= 0.2.9' 'QuickCheck >= 2.4.0.1' 'ChasingBottoms' 'HUnit' 'test-framework-hunit' 'criterion'
+cabal configure -v2 --enable-tests --enable-benchmarks
```

Comment:
You mention several specific dependency versions here. These could easily go out of date. What should we do to make sure they stay in tune with the `.cabal` file?

Reply:
It turns out these specific versions aren't necessary; I've removed them from the instructions.